modules/dnn/src/darknet/darknet_io.cpp

```diff
@@ -191,18 +234,21 @@ namespace cv {
                     fused_layer_names.back() = last_layer;
                 }
 
-                void setMaxpool(size_t kernel, size_t pad, size_t stride)
+                void setMaxpool(int kernel, int pad, int stride)
```

Comment:
Why not to use `size_t`?

Reply:
We can use both of them. I did that because 
https://github.com/opencv/opencv/blob/24932c781ea872cf542536402793b43a8d8c0216/modules/dnn/src/darknet/darknet_io.cpp#L645-L650
here we are taking inputs in `int` type variable.